locations/spiders/ben_and_jerrys.py

```diff
@@ -33,4 +33,9 @@ def parse_item(self, item, location):
                 item["opening_hours"].set_closed(day)
             else:
                 item["opening_hours"].add_ranges_from_string(day + " " + location.get(day.lower()))
+        # this source is not listing standalone ice cream POIs from this brand
+        # but rather known locations where their ice cream is provided
+        item.pop("amenity", None)
```

Comment:
well, this has not worked

Reply:
lets leave idea for review before I will start figuring out why code implementing it is broken (likely NSI matching happens after it, so I need to drop these tags in post_process or stop NSI matching)

Reply:
For reference, see [NSI_MATCHING.md](https://github.com/alltheplaces/alltheplaces/blob/master/docs/NSI_MATCHING.md) for how to disable matching to NSI